### PR TITLE
modify base metadata data structure to match consolidated view and pass run_stats_only flg to super

### PR DESCRIPTION
**Pull request by FICU-tensileai**
## What is this and why are we doing it?
We've made modificiations to the underlying post job metadata data structure to correct the opt_name due to a combining of steps during the  job run itself. The name now matches the view that is being created so it can pick up the proper job information. We've also had to pass the run_stats_only flag all the way to the super runner (TAF_Runner) so we make sure we don't write the post job run to the job_cntl_parms table since it's now separate.

* Link to the Jira ticket for this change: https://jiraent.cms.gov/browse/DBDAART-4766

## What are the security implications from this change?
No security implications here because what we're modifying is confined to metadata job runs and notebooks themselves.

## How did I test this?
https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#job/401450024710359/run/306588641446200
https://cms-dataconnect-val.cloud.databricks.com/?o=955724715920583#notebook/282471729607430/command/1205832889328863

## PR Checklist
- [x] The JIRA ticket number and a short description is in the subject line
- [x] My code follows any applicable [style guides](https://cms-dataconnect.atlassian.net/wiki/search?text=style%20guide)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: taf/TAF_Metadata.py [hunk 1]
```diff
@@ -120,7 +120,7 @@ class TAF_Metadata:
             (493, '012_bsf_ELG00012', '0.1. create_initial_table', 'ELG00012', 'submtg_state_cd', 'submtg_state_cd'),
             (494, '016_bsf_ELG00016', '0.1. create_initial_table', 'ELG00016_{bsf_file_date}_uniq', 'submtg_state_cd', 'submtg_state_cd'),
             (495, '018_bsf_ELG00018', '0.1. create_initial_table', 'ELG00018', 'submtg_state_cd', 'submtg_state_cd'),
-            (498, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_{rpt_out}_{bsf_file_date}', 'submtg_state_cd', 'submtg_state_cd'),
+            (498, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'submtg_state_cd', 'submtg_state_cd'),
```
Comment (RIE9-adhoc) [inline taf/TAF_Metadata.py:123]: It appears after changing this to BSF_STEP1 that there's duplicated lines:
```
            (445, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'distinct msis_ident_num', 'submtg_state_cd'),
            (450, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'submtg_state_cd', 'submtg_state_cd'),
            (477, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'distinct msis_ident_num', 'submtg_state_cd'),
            (498, '023_bsf_ELG00023', '0.1. create_initial_table', 'BSF_STEP1', 'submtg_state_cd', 'submtg_state_cd'),
```
In taf/TAF_Runner.py, `__create_formated_lkup`, the loop goes through all `pgmLkpData` rows, so wouldn't this produce duplicate data?

In the original code there's still a reference to BSF_(rpt_out)_(bsf_file_date):
```
001_batch_bsf.sas: %CREATE_EFTSMETA_INFO(&DA_SCHEMA., &DA_RUN_ID., &TABLE_NAME., 023_bsf_ELG00023, 0.1. create_initial_table, %nrstr(BSF_&RPT_OUT._&BSF_FILE_DATE), submtg_state_cd);
```
Could you explain why this change was made?

Comment (FICU-tensileai) [reply taf/TAF_Metadata.py:123]: Good catch! Yes, I should remove the duplicate lines. Talked to @C1KB-tensileai last week when I discovered what the actual issue was and the BSF_STEP1 combined a couple of steps from the SAS code so that makes sense why there is duplicate now. Let me resolve that.

Comment (FICU-tensileai) [reply taf/TAF_Metadata.py:121]: Done!

**Review by RIE9-adhoc — APPROVED** [review]

**Review by C1KB-tensileai — APPROVED** [review]
Looks good, thanks @FICU-tensileai
